package.json

```diff
@@ -1,10 +1,14 @@
 {
   "name": "marigold",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "description": "A set of common and reusable components and tools for building websites and applications at 18F.",
   "scripts": {
-    "build": "fractal build",
-    "start": "fractal start --sync"
+    "prestart": "npm run clean && npm run build",
+    "postinstall": "source env.sh",
+    "build": "npm run copy-wds-img && fractal build",
+    "clean": "rm -fr build",
+    "start": "fractal start --sync",
+    "copy-wds-img": "cp -r node_modules/uswds/dist/img src/assets/uswds"
```

Comment:
I would expect these to be copied to the `build` folder. I generally don't think it's good practice to copy things into a `src` folder because devs would expect that only to be files they have control of. What's the reason these are copied here? There might need to be an intermediate place for them.